src/features/metadata-based-view/MetadataBasedItemList.js

```diff
@@ -67,18 +75,26 @@ class MetadataBasedItemList extends React.Component<Props, State> {
         };
     }
 
-    getGridCellData(columnIndex: number, rowIndex: number): Element<typeof FileIcon> | string {
-        const { currentCollection, metadataColumnsToShow }: Props = this.props;
-        const { items } = currentCollection;
-        const { name = '', metadata = {} } = items[rowIndex - 1];
+    getGridCellData(columnIndex: number, rowIndex: number): Element<typeof FileIcon | typeof PlainButton> | string {
+        const {
+            currentCollection: { items },
+            metadataColumnsToShow,
+            onItemClick,
+        }: Props = this.props;
+        const item = items[rowIndex - 1];
+        const { name = '', metadata = {} } = item;
```

Comment:
Is the default value for `name` needed here? If `null` or `undefined` is passed to `{name}`, what happens?